sdk/features/arm-features/package.json

```diff
@@ -1,13 +1,19 @@
 {
   "name": "@azure/arm-features",
+  "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
-  "description": "FeatureClient Library with typescript type definitions for node.js and browser.",
-  "version": "1.1.0",
+  "description": "A generated SDK for FeatureClient.",
+  "version": "30.0.0-beta.1",
+  "engines": {
+    "node": ">=12.0.0"
+  },
   "dependencies": {
-    "@azure/ms-rest-azure-js": "^1.4.0",
-    "@azure/ms-rest-js": "^1.11.0",
-    "@azure/core-auth": "^1.1.4",
-    "tslib": "^1.9.3"
+    "@azure/core-paging": "^1.1.1",
+    "@azure/core-client": "^1.0.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.1.0",
+    "tslib": "^2.2.0",
+    "@azure/core-lro": "^2.0.0"
```

Comment:
why was this dep added?

Reply:
This is added manually, because we write a script to keep the dependency version same as other dataplanes'. I think there is a bug and I will fix it later.